dts/bindings/serial/uart-controller.yaml

```diff
@@ -29,3 +29,12 @@ properties:
         - "none"
         - "odd"
         - "even"
+    stop-bits:
+      required: false
+      type: string
+      description: Inital stop-bits setting for UART
```

Comment:
Please add a comment about default value.

Reply:
Adding this to the generic uart-controller binding but only support to the STM32 UART driver will likely lead to user confusion.

Reply:
I guess that all drivers have a default value for this, isn't it ?

Reply:
Sure, but adding a generic devicetree property for this gives the impression that all drivers supports that property. This is not the case with this PR, since it is only adding support for it to the STM32 UART driver.

Reply:
ok, so your advice is:
- Extend this to other serial drivers
or
- Move to STM32 binding 

Correct ?

Reply:
@erwango Correct.

Reply:
I can move it to stm32 binding, but this might make other users not aware of this feature.
On the other hand, I don't think it's correct to ask me to extend this to other drivers since I don't know the implementation details of them. (I only work with stm32, I don't know the 
Which option do you prefer?

Reply:
> Let's move to STM32 binding for now if you're not comfortable with extending the change on other drivers

I agree.